docs/query_sub_system.md

```diff
@@ -0,0 +1,109 @@
+# The Query Sub-System
+
+You can view this document as a prezzo at:
+http://remarkjs.com/remarkise
+
+---
+
+## Caveat Lector
+
+This documentation assumes that you understand Time Series from a user perspective and have read the documentation on it on the main Basho Documentation Site:
+http://docs.basho.com
+
+---
+
+## Overview
+
+Time Series introduces new elements into the supervision tree of `riak_kv`
+
+![Time Series Supervision Tree](https://raw.githubusercontent.com/basho/riak_kv/feature_gg_documents_and_architecture/docs/timeseries_supervision_tree.png)
```

Comment:
Suggest emphasizing somehow in the image that the `w1c` tree is relevant but *not* new to TS. Perhaps a different color for the elements in the tree.